beacon-chain/validator-monitor/process_attestation.go

```diff
@@ -0,0 +1,165 @@
+package validator_monitor
```

Comment:
```suggestion
package monitor
```

`monitor` for simplicity and better readability imo. I think @kasey will like that better